### PR TITLE
Change style options and add default case

### LINKED ISSUE #54: Give actual Style Options in drop-down menu
Currently in the dropdown menu for the styles, the style options are "Style 1", "Style 2", ...
We should change that, so it shows the actual styles we can offer.

Additionally, we should also handle the case, that the user does not choose a style in the run function in t2iadapter.py
Currently, we have no such default case

### PR DESCRIPTION
**Pull request by TamedNightFury**
Resolves #54 (hopefully)

I made the following changes:
-In index.html I changed the definitions of the buttons in the dropdown menu to show the names of our style Options, and added more buttons for all of the options
-In script.js I added onto the changeStyle function in the switch case, so all of the buttons can be selected
After that, we read the selected style in the generate function in script.js and give that to t2iadapter.py where we match that style and give it to the run function. So I believe the style options should work like that and not need other changes

For the default case of no style, I added a catch case in the match function in t2iadapter.py
Then I added an if clause in front of the line where we load the style model.
So if none of our style options are selected, none of the style models are loaded, and the image should be generated without a special style (just like we did at the beginning)

Sadly I'm not entirely sure how I can test if this works, so it would be great if someone could try this out and see if any errors occur

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -43,9 +43,12 @@ <h1 id="headline">Pencil<span style="color: brown">2</span>Pixel</h1>
             <button id="dropbtn">Select Style</button>
             <div id="dropdown-content">
                 <a id="nostyle" onclick="changeStyle(0)">No Style</a>
-                <a id="style1" onclick="changeStyle(1)">Style 1</a>
-                <a id="style2" onclick="changeStyle(2)">Style 2</a>
-                <a id="style3" onclick="changeStyle(3)">Style 3</a>
+                <a id="style1" onclick="changeStyle(1)">Photography</a>
```
Comment (EveEme) [inline frontend/index.html]: You could change the id name, so it's clear which style is passed in the switch case in script.js

**Review by EveEme — APPROVED** [review]
Overall it looks good. :)

File: frontend/index.html (hunk 1 above)
Comment (TamedNightFury) [reply frontend/index.html]: Makes sense, I changed the id names to their corresponding style

**Comment by EveEme** [comment]
I tested it on the server. The style option "crayon" doesn't work. If i choose "crayon" i get an uncaught error.

**Comment by TamedNightFury** [comment]
Ah, there was a typo in the script.js file on the server. Does it work now?

**Pull request merged**